resistance-is-futile/usr/share/asteroid-launcher/watchfaces/resistance-is-futile.qml

```diff
@@ -0,0 +1,157 @@
+                    pixelSize: parent.height * 0.31
+                }
+                color: "white"
+                horizontalAlignment: Text.AlignHCenter
+                anchors {
+                    centerIn: parent
+                    verticalCenterOffset: -parent.height * 0.005
+                }
+                text: "<b>AsteroidOS</b><br>Hack Your Wrist"
+            }
+        }
+        MouseArea {
+            anchors.fill: parent
+            onPressAndHold: asteroidLogo.visible ?
+                                asteroidLogo.visible = false :
+                                asteroidLogo.visible = true
+        }
+    }
+
+    Text {
+        id: datetext
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: parent.height * 0.25 
+        }
+        font {
+            family: "Titillium"
+            weight: Font.Thin
+            pixelSize: parent.height * 0.06
+        }
+        color: "white"
+        text: wallClock.time.toLocaleString(Qt.locale(), "ddd dd");
+    }
+
+    Rectangle {
+        id: time
+        color: displayAmbient ? "#371102" : "#6e2304"
+        anchors.centerIn: parent
+        height: parent.height * 0.30
+        width: parent.width
+        Repeater{
+            id: digits
+            model: 4
+            Rectangle {
+                color: ["black","brown","red","orange","yellow","green","blue","violet","gray","white"][getTimeDigit(wallClock.time, index)]
+                opacity: displayAmbient ? 0.7 : 1
+                height: parent.height
+                width: parent.width * 0.1
+                x: parent.width / 5 * index + parent.width / 10
+            }
+        }
+    }
+
+    Text {
+        id: ampm
+        visible: use12H.value
+        anchors {
+            centerIn: parent
+            horizontalCenterOffset: parent.width * 0.4
+        }
+        font {
+            family: "Titillium"
+            weight: Font.Bold
+            pixelSize: parent.height * 0.05
+        }
+        color: "white"
+        text: wallClock.time.toLocaleString(Qt.locale(), "ap");
+    }
+
+    Repeater{
+        id: secondMarks
+        model: 60
+        Rectangle {
+            id: second
+            visible: !displayAmbient
+            antialiasing : true
+            color: "transparent"
+            width: parent.width * 0.01
+            height: parent.height * 0.125
+            transform: [
+                Rotation { 
+                    origin.x: second.width / 2
+                    origin.y: parent.height * 0.48
+                    angle: (index) * 360 / secondMarks.count 
+                },
+                Translate { 
+                    x: (parent.width - second.width) / 2
+                    y: parent.height / 2 - parent.height * 0.48
+                }
+            ]
+            Rectangle {
+                anchors.centerIn: parent
+                anchors.fill: parent
+                opacity: wallClock.time.getSeconds() == index ? 1 : 0
```

Comment:
Given that only one is shown at a time, how about using only one Rectangle (with a transform: that uses wallClock.time.getSeconds() rather than index ?

Reply:
I decided to change entirely to use a resistor symbol svg